### PR TITLE
[CP Staging] Fix icons after regression

### PR DESCRIPTION
**Pull request by narefyev91**
### Details
Fix button for distance and quick actions for messages

### Fixed Issues

$ https://github.com/Expensify/App/issues/38266
$ https://github.com/Expensify/App/issues/38264
$ https://github.com/Expensify/App/issues/38284
$ https://github.com/Expensify/App/issues/38291
PROPOSAL:

### Tests
Checking quick actions:
1. Go to any chat
2. Send a message
3. Hover over the message for the action menu to appear

Checking distance button:
1. Go to https://staging.new.expensify.com/
2. Tap fab ---Request money--Distance
3. Enter start and stop point
4. Note Add stop button

Checking rotate button:
1. Access staging.new.expensify.com
2. Sign into any account
3. Go to Profile Pic or WS Pic and tap on Upload
4. Observe the rotate button

- [x] Verify that no errors appear in the JS console

### Offline tests
Checking quick actions:
1. Go to any chat
2. Send a message
3. Hover over the message for the action menu to appear

Checking distance button:
1. Go to https://staging.new.expensify.com/
2. Tap fab ---Request money--Distance
3. Enter start and stop point
4. Note Add stop button

Checking rotate button:
1. Access staging.new.expensify.com
2. Sign into any account
3. Go to Profile Pic or WS Pic and tap on Upload
4. Observe the rotate button

### QA Steps
Checking quick actions:
1. Go to any chat
2. Send a message
3. Hover over the message for the action menu to appear

Checking distance button:
1. Go to https://staging.new.expensify.com/
2. Tap fab ---Request money--Distance
3. Enter start and stop point
4. Note Add stop button

Checking rotate button:
1. Access staging.new.expensify.com
2. Sign into any account
3. Go to Profile Pic or WS Pic and tap on Upload
4. Observe the rotate button

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

android1
android2
android

</details>

<details>
<summary>Android: mWeb Chrome</summary>

android-web
android-web2
android-web3

</details>

<details>
<summary>iOS: Native</summary>

ios1
ios2
ios4

</details>

<details>
<summary>iOS: mWeb Safari</summary>

ios-web1
ios-web2
ios-web4

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

web1
web2
Screenshot 2024-03-14 at 15 10 39

</details>

<details>
<summary>MacOS: Desktop</summary>

desktop1
desktop2
Screenshot 2024-03-14 at 15 10 39

</details>

### CODE REVIEW

**Comment by narefyev91** [comment]
@Expensify/design @shawnborton please take a look:
Quick actions - looks now the same as on prod
Distance button - updated alignment

**Review by MonilBhavsar — COMMENTED** [review]
The rotate icon svg is 16x16. Should we update it to 20x20?

**Comment by narefyev91** [comment]
> The rotate icon svg is 16x16. Should we update it to 20x20?

I think no - button is specified as medium - and icon should be medium as well.

**Comment by narefyev91** [comment]
> Thanks for the fixes @narefyev91 !
> 
> I agree that in default sized buttons that also have a label, we want the icon to be 16x16.
> 
> However, if the button **only** has an icon, it looks like the icon should be 20x20. Does that sound right @Expensify/design ?

Ohh yup you right - double checked other icons - and in default state we using 20x20 - just made changes for this current one

**Comment by MonilBhavsar** [comment]
The dimensions of button are different somehow. I think we want them as a common ratio?
### Dev(this branch)
Screenshot 2024-03-14 at 6 58 46 PM
### Production
Screenshot 2024-03-14 at 6 59 14 PM

**Comment by shawnborton** [comment]
I actually think ideally, that button would be perfectly round at 40x40. Basically that's what we want to do for "icon buttons" - a button that just has an icon and no label. It should be perfectly round, and at the 40x40 size, the icon should be 20x20.

**Comment by MonilBhavsar** [comment]
That would be good. Thanks Shawn for the thought!

**Comment by dannymcclain** [comment]
Agree with everything Shawn said! We tread our "icon buttons" a little differently than regular buttons. Tried to whip up some quick specs real fast:

image

**Comment by shawnborton** [comment]
<3 LOVE IT

**Comment by narefyev91** [comment]
bank account custom button - moves back to prod view
Screenshot 2024-03-14 at 15 20 36

**Comment by shawnborton** [comment]
Looks like it's slightly off still:
CleanShot 2024-03-14 at 10 16 15@2x

**Comment by narefyev91** [comment]
> Looks like it's slightly off still: CleanShot 2024-03-14 at 10 16 15@2x

but the same is on the prod

**Comment by shawnborton** [comment]
Got it, let's fix it then. We shouldn't touch the non-green option row, just make the green option row match what is below it.

**Comment by dannymcclain** [comment]
Looks like the icons are not aligned the same either. 😬

CleanShot 2024-03-14 at 09 23 37@2x

**Comment by shawnborton** [comment]
> updated button component to support styles for just only icon passed

Those look good to me.

**Comment by s77rt** [comment]
quick action icons not same size
Screenshot 2024-03-14 at 3 44 19 PM
Screenshot 2024-03-14 at 3 44 25 PM
Screenshot 2024-03-14 at 3 44 51 PM

**Comment by narefyev91** [comment]
> quick action icons not same size Screenshot 2024-03-14 at 3 44 19 PM Screenshot 2024-03-14 at 3 44 25 PM Screenshot 2024-03-14 at 3 44 51 PM

@shawnborton which size should be this icons?

**Comment by s77rt** [comment]
Workspace tags settings button have different icon size compared to production, is this intended?

Screenshot 2024-03-14 at 3 52 05 PM
Screenshot 2024-03-14 at 3 52 07 PM

**Comment by s77rt** [comment]
Workspace tags and categories settings button are diff (spacing after icon)

Screenshot 2024-03-14 at 3 53 11 PM

Screenshot 2024-03-14 at 3 53 09 PM

**Comment by dannymcclain** [comment]
>quick action icons not same size

In Figma they are `18x18` but is that right? I mean... it's not base 4... cc @shawnborton @dubielzyk-expensify

**Comment by narefyev91** [comment]
> Workspace taxes and tags settings buttons are diff (spacing after icon)
> 
> Screenshot 2024-03-14 at 3 53 11 PM Screenshot 2024-03-14 at 3 53 09 PM

correct spaces should be these
Screenshot 2024-03-14 at 17 25 52

**Comment by s77rt** [comment]
Profile email copy button diff size compared to production
Screenshot 2024-03-14 at 4 25 30 PM
Screenshot 2024-03-14 at 4 25 26 PM

**Comment by shawnborton** [comment]
> Workspace tags settings button have different icon size compared to production, is this intended?

The icon size looks good. 

> Workspace taxes and tags settings buttons are diff (spacing after icon)

Yeah, I'm not sure why there is more space between the buttons but the buttons themselves look good. 

> In Figma they are 18x18 but is that right? I mean... it's not base 4... cc @shawnborton @dubielzyk-expensify

What are they currently on production, not staging?

**Comment by narefyev91** [comment]
> Profile email copy button diff size compared to production Screenshot 2024-03-14 at 4 25 30 PM Screenshot 2024-03-14 at 4 25 26 PM

Fixed
Screenshot 2024-03-14 at 17 43 43

File: src/components/DistanceRequest/DistanceRequestFooter.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ function DistanceRequestFooter({waypoints, transaction, mapboxAccessToken, navig
                         onPress={() => navigateToWaypointEditPage(Object.keys(transaction?.comment?.waypoints ?? {}).length)}
                         text={translate('distance.addStop')}
                         isDisabled={numberOfWaypoints === MAX_WAYPOINTS}
-                        innerStyles={[styles.ph10]}
+                        innerStyles={[styles.pl10, styles.pr10]}
```
Comment (s77rt) [inline src/components/DistanceRequest/DistanceRequestFooter.tsx:99]: Unnecessary change?

File: src/styles/utils/index.ts [hunk 2]
```diff
@@ -499,6 +499,10 @@ function getButtonStyleWithIcon(
             return useDefaultButtonStyles ? styles.buttonLarge : {...styles.buttonLarge, ...(hasText ? verticalStyle : styles.ph0)};
         }
         default: {
+            if (hasIcon && !hasText) {
+                return {...styles.buttonMedium, ...styles.ph0};
+            }
```
Comment (s77rt) [inline src/styles/utils/index.ts:504]: What case is this fixing?

File: src/styles/utils/sizing.ts [hunk 3]
```diff
@@ -14,6 +14,10 @@ export default {
         height: 272,
     },
 
+    h52: {
+        height: 52,
+    },
+
```
Comment (s77rt) [inline src/styles/utils/sizing.ts:20]: This does not belong here as it does not respect the base 4 rule (h52 should be 208)

File: src/styles/utils/spacing.ts [hunk 4]
```diff
@@ -119,6 +119,10 @@ export default {
         marginRight: 20,
     },
 
+    mr23: {
+        marginRight: 23,
+    },
+
```
Comment (s77rt) [inline src/styles/utils/spacing.ts:125]: Same ^

**Review by s77rt — CHANGES_REQUESTED** [review]

**Comment by s77rt** [comment]
@luacmartins Yes

File: src/components/DistanceRequest/DistanceRequestFooter.tsx (hunk 1 above)
Comment (narefyev91) [reply src/components/DistanceRequest/DistanceRequestFooter.tsx:99]: @s77rt please check this one https://github.com/Expensify/App/issues/38266#issuecomment-1996757522

File: src/styles/utils/index.ts (hunk 2 above)
Comment (narefyev91) [reply src/styles/utils/index.ts:504]: @s77rt this one https://github.com/Expensify/App/pull/38275#issuecomment-1997512298

File: src/styles/utils/sizing.ts (hunk 3 above)
Comment (narefyev91) [reply src/styles/utils/sizing.ts:20]: fixed

File: src/styles/utils/spacing.ts (hunk 4 above)
Comment (shawnborton) [reply src/styles/utils/spacing.ts]: Also, are we sure that shouldn't be 24? We basically always use base4 for all of our numeric values.

**Comment by s77rt** [comment]
## Reviewer Checklist

- [X] I have verified the author checklist is complete (all boxes are checked off).
- [X] I verified the correct issue is linked in the `### Fixed Issues` section above
- [X] I verified testing steps are clear and they cover the changes made in this PR
    - [X] I verified the steps for local testing are in the `Tests` section
    - [X] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [X] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [X] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I verified tests pass on **all platforms** & I tested again on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [X] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] If a new component is created I verified that:
    - [X] A similar component doesn't exist in the codebase
    - [X] All props are defined accurately and each prop has a `/** comment above it */`
    - [X] The file is named correctly
    - [X] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [X] The only data being stored in the state is data necessary for rendering and nothing else
    - [X] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [X] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [X] All JSX used for rendering exists in the render method
    - [X] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [X] I verified that all the inputs inside a form are aligned with each other.
    - [X] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [X] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>


android

</details>

<details>
<summary>Android: mWeb Chrome</summary>


mweb-chrome

</details>

<details>
<summary>iOS: Native</summary>


ios

</details>

<details>
<summary>iOS: mWeb Safari</summary>


mweb-safari

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>


web

</details>

<details>
<summary>MacOS: Desktop</summary>


desktop

</details>

File: src/styles/utils/spacing.ts (hunk 4 above)
Comment (narefyev91) [reply src/styles/utils/spacing.ts]: yup - 24 will not be pixel perfect
this is 24
@shawnborton 
Screenshot 2024-03-14 at 18 51 13

Comment (narefyev91) [reply src/styles/utils/spacing.ts]: Seems 24 is in prod as well - but text will be not on the same line. If we ok with that i can switch to 24

Comment (shawnborton) [reply src/styles/utils/spacing.ts]: Hmm I guess I'm just trying to understand how 23 actually works and 24 doesn't... where is this 1px coming from? But, given that this is a one-off button style, I can live with it and hopefully some day we revisit all of our row styles.

**Comment by s77rt** [comment]
@narefyev91 Can you please fix this inconsistency https://github.com/Expensify/App/pull/38275#issuecomment-1997646219 should be a quick fix, there is an extra `iconStyles={[styles.mr2]}` in WorkspaceTagsPage and WorkspaceCategoriesPage

**Comment by s77rt** [comment]
One more inconsistency in the spacing between the buttons, some use `styles.mr3` others user `styles.pr2`
Screenshot 2024-03-14 at 5 58 06 PM
Screenshot 2024-03-14 at 5 58 10 PM


What's the correct spacing here? @Expensify/design

**Comment by narefyev91** [comment]
> WorkspaceCategoriesPage

done

File: src/components/DistanceRequest/DistanceRequestFooter.tsx (hunk 1 above)
Comment (s77rt) [reply src/components/DistanceRequest/DistanceRequestFooter.tsx:99]: Interesting!

**Comment by s77rt** [comment]
@narefyev91 Push the changes 😅

**Comment by narefyev91** [comment]
> @narefyev91 Push the changes 😅

lol - i already pushed them

**Comment by s77rt** [comment]
Github UI tricked me 😂

**Review by s77rt — APPROVED** [review]
Looks good but let's hold merge until this is resolved https://github.com/Expensify/App/pull/38275#issuecomment-1997922389

**Comment by narefyev91** [comment]
> One more inconsistency in the spacing between the buttons, some use `styles.mr3` others use `styles.pr2` Screenshot 2024-03-14 at 5 58 06 PM Screenshot 2024-03-14 at 5 58 10 PM
> 
> What's the correct spacing here? @Expensify/design

cc @shawnborton can you please give correct spacing here

**Comment by narefyev91** [comment]
> It depends on the size of the button, but here are the specs:
> 
> image

@dannymcclain we talking about gap between 2 buttons Button1 and Button2 - how much space should be between them?

**Comment by narefyev91** [comment]
@dannymcclain https://github.com/Expensify/App/pull/38275#issuecomment-1997950248 - you see that on both screens - gap between Add rate and settings is not the same as between Add tag and settings. One of them has incorrect margin/padding. We need to know - which one is correct

**Comment by dannymcclain** [comment]
@narefyev91 LOL my bad!! I totally see what you're talking about now 😂

The correct spacing is `12px` as far as I know.

**Comment by narefyev91** [comment]
Screenshot 2024-03-14 at 19 36 54
should be 12 px between buttons

**Comment by s77rt** [comment]
@luacmartins This is ready for merge

**Review by luacmartins — APPROVED** [review]

**Comment by mountiny** [comment]
Not emergency

**Comment by luacmartins** [comment]
Tests were passing
Screenshot 2024-03-14 at 11 51 42 AM
